pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "maturin"
 [project]
 name = "rf24-py"
 description = "A python package binding the nRF24/rf24-rs rust project."
-requires-python = ">=3.8"
+requires-python = ">=3.9"
```

Comment:
I had to bump the minimum required python version because `uv` found a docs dependency that does not support with python v3.8.

Subsequently, I also updated the license metadata to adopt [PEP639](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files) which satisfies install-time warnings (see nRF24/pyRF24#105 for similar detail).